editor/dependency_editor.cpp

```diff
@@ -757,7 +757,33 @@ void DependencyErrorDialog::ok_pressed() {
 }
 
 void DependencyErrorDialog::custom_action(const String &) {
-	EditorNode::get_singleton()->fix_dependencies(for_file);
+	EditorNode::get_singleton()->fix_dependencies( for_file, std::bind(&DependencyErrorDialog::on_update_callback, this, std::placeholders::_1) );
```

Comment:
The stl [is not allowed](https://docs.godotengine.org/en/latest/contributing/development/cpp_usage_guidelines.html#standard-template-library) in general, and this can be accomplished with our own `Callable` class it looks like

Reply:
Hi, me and @CampioniMan are contributing for the project as a homework for university. We have noticed this but haven't figured out how to implement it using the Callable class yet. Just created the PR for now so we can show it for the final presentation. We are still going to rewrite this implementation using the correct classes.

Reply:
Then please mark this as a draft, and clarify these details in the PR, that's really important information to add